### PR TITLE
[PROGRAMMES-7164] Update PHP to 7.3

### PR DESCRIPTION
**Pull request by benolot**
Ticket: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-7164

### CODE REVIEW

File: script/codebuild/build [hunk 1]
```diff
@@ -10,6 +10,8 @@ set -e
 # INSTALL #
 ###########
 
+php --version
```
Comment (BjornTwachtmann) [inline script/codebuild/build]: did you mean to leave this in?

Comment (benolot) [reply script/codebuild/build]: Don't see much harm in leaving it in for future debugging

File: script/codebuild/build [hunk 2]
```diff
@@ -10,6 +10,8 @@ set -e
...
+
 # Fail if we're not using php71+
 if ! (php --version | grep -q "PHP 7.") ; then
     echo "You're not using PHP 7.1+, please install it."
```
Comment (Tasiobg) [inline script/codebuild/build]:
```suggestion
    echo "You're not using PHP 7.3+, please install it."
```

File: script/codebuild/build [hunk 3]
```diff
@@ -10,6 +10,8 @@ set -e
 # INSTALL #
 ###########
 
+php --version
+
 # Fail if we're not using php71+
```
Comment (Tasiobg) [inline script/codebuild/build]:
```suggestion
# Fail if we're not using php73+
```

**Pull request merged**